vpr/src/route/spatial_route_tree_lookup.cpp

```diff
@@ -17,7 +17,8 @@ SpatialRouteTreeLookup build_route_tree_spatial_lookup(const Netlist<>& net_list
     float bb_area_per_sink = bb_area / fanout;
     float bin_area = BIN_AREA_PER_SINK_FACTOR * bb_area_per_sink;
 
-    float bin_dim = std::ceil(std::sqrt(bin_area));
+    constexpr float MIN_BIN_DIM = 3;
```

Comment:
Comment what this is and why we need it.  Too small and the high fanout sampling doesn't work well.